R/conjugate_encoder.R

```diff
@@ -36,6 +36,8 @@ library(rlang)
 #' objective = "regression")
 conjugate_encoder <- function(X_train, X_test = NA, y, cat_columns, prior_params, objective = "regression") {
```

Comment:
Inconsistency with my code in that you used X_test=NA instead of X_test = NULL, not sure if changing it will mess up another part of your code. 
```suggestion
conjugate_encoder <- function(X_train, X_test = NULL, y, cat_columns, prior_params, objective = "regression") {
```

Reply:
I agree! Don't know what I thinking there, typically it's always assigned NULL like you suggested. Thanks!